Python/ceval.c

```diff
@@ -786,7 +786,11 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
 
     _PyInterpreterFrame  entry_frame;
 
-
+    if (_Py_EnterRecursiveCallTstate(tstate, "")) {
+        assert(frame->owner != FRAME_OWNED_BY_INTERPRETER);
```

Comment:
Don't you need to `_Py_LeaveRecursiveCallPy` after this as well?

At least from reading the code of `exit_unwind`.

Reply:
No. We haven't called. `_Py_EnterecursiveCallPy`